### PR TITLE
HPC Installation Instructions

### PR DESCRIPTION
**Pull request by Andesha**
The following is my workflow for getting everything running smoothly in an environment on Narval.

Hopefully addresses #126 and #98 for the time being.

### CODE REVIEW

**Comment by scott-huberty** [comment]
Thanks @Andesha - I will definitely give this a swing on Narval this week!

Re: documentation, I wonder if we should put this as a dedicated page in our documentation, specifically for CC users, since these directions are specific to compute canada (i.e. other HPC's won't necessarily have a wheel built for mne?). 

I could do that but it will probably be a while before it gets done.

File: README.md [hunk 1]
```diff
@@ -56,6 +56,39 @@ Once you are ready, you can save your file:
...
+
+pip install --no-index mne
+pip install --no-index pandas
+pip install --no-index xarray
+pip install --no-index pyyaml
+pip install --no-index sklearn
+pip install mne_bids
+
+# Clone down mne-iclabel and switch to the right version and install it locally
+git clone https://github.com/mne-tools/mne-icalabel.git
```
Comment (scott-huberty) [inline README.md:78]: Out of curiosity - is it necessary to checkout a certain branch of mne-icalabel?

**Comment by scott-huberty** [comment]
Just tried out your instructions. Runs on narval like a charm! Even got a 50% speed improvement compared to my local computer.

Awesome that we have a wheel for MNE on compute canada now.

File: README.md [hunk 2]
```diff
@@ -56,6 +56,39 @@ Once you are ready, you can save your file:
...
+virtualenv --no-download eeg-env
+source eeg-env/bin/activate
+
+pip install --no-index mne
+pip install --no-index pandas
+pip install --no-index xarray
+pip install --no-index pyyaml
+pip install --no-index sklearn
+pip install mne_bids
+
```
Comment (scott-huberty) [inline README.md:76]: the user also needs to `pip install edflib-python` for full lossless functionality (saving).

**Comment by Andesha** [comment]
Feel free to make the change or move it into a wiki page or something - any of those sorts of things are fine with me

**Comment by christian-oreilly** [comment]
> Re: documentation, I wonder if we should put this as a dedicated page in our documentation, specifically for CC users, since these directions are specific to compute canada (i.e. other HPC's won't necessarily have a wheel built for men?).

Indeed, a specific tutorial page seems a good idea. +1

**Review by scott-huberty — APPROVED** [review]

**Pull request merged**